idr0045-study.txt

```diff
@@ -0,0 +1,83 @@
+# STUDY DESCRIPTION SECTION				
+Comment[IDR Study Accession]	idr0045			
+Study Title	Dual spindle formation in zygotes keeps parental genomes apart in early mammalian embryos			
+Study Description	At the beginning of mammalian life the genetic material from each parent meets when the fertilized egg divides. It was previously thought that a single microtubule spindle is responsible to spatially combine the two genomes and then segregate them to create the two-cell embryo. Utilizing light-sheet microscopy, we showed that two bipolar spindles form in the zygote, which independently congress the maternal and paternal genomes. These two spindles aligned their poles prior to anaphase but kept the parental genomes apart during the first cleavage. This spindle assembly mechanism provides a potential rationale for erroneous divisions into more than two blastomeric nuclei observed in mammalian zygotes and reveals the mechanism behind the observation that parental genomes occupy separate nuclear compartments in the two-cell embryo.			
+Study Organism	Mus musculus	Mus musculus x Mus spretus		
+Study Organism Term Source REF	NCBITaxon			
+Study Organism Term Accession	10090	186842		
+Study Experiments Number	1			
+Study Type	5D light sheet and confocal microscopy imaging of mouse zygotes			
+				
+# Study Publication				
+Study PubMed ID	30002254			
+Study Publication Title	Dual spindle formation in zygotes keeps parental genomes apart in early mammalian embryos			
+Study Author List	Judith Reichmann, Bianca Nijmeijer, M. Julius Hossain, Manuel Eguren, Isabell Schneider, Antonio Z. Politi, Maria J. Roberti, Lars Hufnagel, Takashi Hiiragi and Jan Ellenberg			
+Study DOI	https://doi.org/10.1126/science.aar7462			
+Study Copyright	Reichmann et al			
+				
+# Study Contacts	Ellenberg			
+Study Person Last Name	Jan			
+Study Person First Name	jan.ellenberg@embl.de			
+Study Person Email	EMBL Heidelberg, Meyerhofstrasse 1, 69117 Heidelberg, Germany EMBL Heidelberg, Meyerhofstrasse 1, 69117 Heidelberg, Germany			
+Study Person Address	principal investigator	submitter		
+Study Person Roles				
+				
+# Study License and Data DOI	CC-BY 4.0			
+Study License	https://creativecommons.org/licenses/by/4.0/			
+Study License URL	University of Dundee			
+Study Data Publisher	https://doi.org/10.17867/10000115			
+Study Data DOI				
```

Comment:
Looks like the values and the keys were shifted by one row. Also  add `Study Copyright\tReichman et al`?